repository/src/main/resources/alfresco/subsystems/Search/noindex/noindex-search-context.xml

```diff
@@ -99,7 +99,13 @@
             <ref bean="search.indexerAndSearcherFactory" />
         </property>
     </bean>
-    
+
+    <bean id="search.fts.alfresco.switching" parent="base.search.fts.alfresco.switching" class="org.alfresco.repo.search.impl.DbOrIndexSwitchingQueryLanguage" >

```

Comment:
Maybe it will be better to define class type `class="org.alfresco.repo.search.impl.DbOrIndexSwitchingQueryLanguage"` within (abstract) bean parent. It won't be required to repeat it so many times.

Reply:
Ah, I think I know what you mean. Good idea if it works - I'll try it 🤞

Reply:
@krdabrowski I've incorporated your suggestion. Thanks 👍 

Marking this thread as resolved and will ask you for a re-review.